app/models/chf/env.rb

```diff
@@ -139,18 +139,24 @@ def self.instance
     define_key :app_role
     define_key :service_level
 
-    define_key :use_image_server_on_show_page,
-      system_env_transform: BOOLEAN_TRANSFORM,
-      default: -> { false }
-
-    define_key :use_image_server_on_viewer,
-      system_env_transform: BOOLEAN_TRANSFORM,
-      default: -> { false }
-
-    define_key :use_image_server_downloads,
-      system_env_transform: BOOLEAN_TRANSFORM,
-      default: -> { false }
-
+    # should be a recognized image service type, or nil/false for none, just hydra-derivatives
```

Comment:
added link to code that has recognized values, to avoid comment becoming out of sync.